storybook/storybook-react/src/components/grid/PageGrid.stories.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { GridCell, PageGrid, Paragraph } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/grid/page-grid.scss'
+import '@amsterdam/design-system-css/src/grid/grid-cell.scss'
+import '@amsterdam/design-system-css/src/heading/heading.scss'
+import '@amsterdam/design-system-css/src/paragraph/paragraph.scss'
+import '../../../../storybook-css/src/docs/stories.scss'
+
+const meta = {
+  title: 'PageGrid',
+  id: 'page-grid',
+  component: PageGrid,
+} satisfies Meta<typeof PageGrid>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
```

Comment:
I don't think this fragment is necessary, Storybook doesn't render it well either

Reply:
Should work but it doesn’t? Could you try next week?